storage/xtradb/handler/ha_innodb.cc

```diff
@@ -11025,6 +11064,22 @@ create_table_check_doc_id_col(
 	return(false);
 }
 
+/*********************************************************************//*
+ creates a column name for hash index */
+UNIV_INTERN
+char*
+innobase_create_column_name_for_hash_index(
+	/*=====================================*/
+	mem_heap_t*		heap,	    /*!< in: memory heap */
+	const char*		index_name)	/*!< in: name of index */
+{
+	ulint len = strlen("HASH_COL_");
+	ulint len1 = strlen("index_name");
```

Comment:
Why you take length of constant string "index_name" here instead of parameter index_name string length ?